cpp/oneapi/dal/algo/svm/test/serialization.cpp

```diff
@@ -241,4 +247,17 @@ TEMPLATE_LIST_TEST_M(svm_serialization_test,
     this->run_test();
 }
 
+TEMPLATE_LIST_TEST_M(svm_serialization_test,
```

Comment:
why not merged with the classification?

Reply:
Because SMO method is used in classification tests, nuSVC doesn't compile with it